object_detection/scripts/coord_pos.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+
+
+# import Locally used packages
+from math import cos, sin, radians
+
+class CoordPositionClass():
```

Comment:
I think naming this "CoordPosition" is better, as the "Class" part doesn't add anything